src/main/java/org/jabref/logic/msbib/MSBibEntry.java

```diff
@@ -262,15 +244,16 @@ public Element getEntryDom(Document document) {
             addField(document, rootNode, entry.getKey(), entry.getValue());
         }
 
-        // based on bibtex content
-        if (dateAccessed != null) {
-            Matcher matcher = DATE_PATTERN.matcher(dateAccessed);
-            if (matcher.matches() && (matcher.groupCount() >= 3)) {
-                addField(document, rootNode, "Month" + "Accessed", matcher.group(1));
-                addField(document, rootNode, "Day" + "Accessed", matcher.group(2));
-                addField(document, rootNode, "Year" + "Accessed", matcher.group(3));
-            }
-        }
+        Optional<Date> parsedDateAcesseField = Date.parse(dateAccessed);
+        String yearAccessed = parsedDateAcesseField.flatMap(Date::getYear).map(accYear -> accYear.toString())
+                .orElse(null);
```

Comment:
The null values are an indicator that no value is present, and in this case it is consistent to the rest of the code. In the addField method there is already a  check ` if( value == null ) return `